docs/source/tutorials/knn_outlier.ipynb

```diff
@@ -0,0 +1,447 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c1f65acd",
+   "metadata": {},
+   "source": [
+    "# Detecting Outliers with Cleanlab and Sklearn\n",
+    "\n",
+    "This 5-minute quickstart tutorial shows how to detect potential outliers in image classification data. The dataset used is `cifar10` which contains 60,000 images. Each image belongs to 1 of 10 categories: airplane, automobile, bird, cat, deer, dog, frog, horse, ship, truck. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:**\n",
+    "\n",
+    "- Load the [cifar10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset and do some basic data pre-processing.\n",
+    "- Create `trainX` and `testX` such that `testX` contains extra categories\n",
+    "- Load pretrained model and extract feature embeddings of `trainX` and `testX`\n",
+    "- Compute outlier scores for each example using cleanlab's `get_outlier_scores` method and analyze results."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7e987fa",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies\n",
+    "You can use `pip` to install all packages required for this tutorial as follows:\n",
+    "\n",
+    "```ipython3\n",
+    "!pip install torch\n",
+    "!pip install cleanlab\n",
+    "...\n",
+    "# Make sure to install the version corresponding to this tutorial\n",
+    "# E.g. if viewing master branch documentation:\n",
+    "#     !pip install git+https://github.com/cleanlab/cleanlab.git\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7b552a12",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Package installation (hidden on docs website).\n",
+    "# If running on Colab, may want to use GPU (select: Runtime > Change runtime type > Hardware accelerator > GPU)\n",
+    "\n",
+    "dependencies = [\"cleanlab\", \"matplotlib\", \"torch\", \"pylab\", \"keras\", \"sklearn\", \"timm\", ]\n",
+    "\n",
```

Comment:
Sometimes these tutorials can break if the dependency packages get updated in some way that is not compatible with the other dependencies.

So from now on, let's start keeping track of which exact package versions the author of tutorial is using.  Can you add a comment here:

```
# package versions we used: matplot==X.X.X, torch==Y.Y.Y, ...
```